server/prediction.py

```diff
@@ -0,0 +1,10 @@
+def predict():
```

Comment:
This looks awful :-) Maybe there is something similar to Scala? If not then it would be better to create list of lines

And return doesn't need braces.